Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -160,5 +204,64 @@ public TestConductRepository(TrappistDbContext dbContext)
             return report.TestStatus;
         }
         #endregion
+
+        #region Private Method
+
+        private async Task TransformAttendeeAnswer(int attendeeId)
+        {
+            var attendeeAnswer = await _dbContext.AttendeeAnswers.SingleOrDefaultAsync(x => x.Id == attendeeId);
+
+            if(attendeeAnswer != null)
+            {
+                if(attendeeAnswer.Answers != null)
+                {
+                    var deserializedAnswer = JsonConvert.DeserializeObject<TestAnswerAC[]>(attendeeAnswer.Answers).ToList();
+                    
+                    foreach(var answer in deserializedAnswer)
+                    {
+                        var testAnswers = new TestAnswers();
+                        //Adding attempted Question to TestConduct table
+                        var testConduct = new DomainModel.Models.TestConduct.TestConduct()
+                        {
+                            QuestionId = answer.QuestionId,
+                            QuestionStatus = answer.QuestionStatus,
+                            TestAttendeeId = attendeeId
+                        };
+                        await _dbContext.TestConduct.AddAsync(testConduct);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1066)